resources/templates/ha-vpn-interop-guide-template/index.md

```diff
@@ -0,0 +1,836 @@
+are some examples of IKE algorithms to specify as part of the instructions.>
+
+-  **Encryption algorithm**—< list required algorithms here>
+-  **Integrity algorithm**—< list required algorithms here>
+-  **Diffie-Hellman group—**< list required group here>
+
+```
+<insert configuration code snippet here>
+```
+
+#### Configure the IKEv2 keyring
+
+< insert the instructions for creating the IKEv2 keyring here.>
+
+```
+<insert configuration code snippet here>
+```
+
+#### Configure the IKEv2 profile
+
+< insert the instructions for creating the IKEv2 profile here.>
+
+```
+<insert configuration code snippet here>
+```
+
+#### Configure the IPsec security association (SA)
+
+< insert the instructions for creating the IPsec SA here. Below is an example of
+parameters to set.>
+
+-  **IPsec SA replay window-size**—Set this to 1024, which is the
+recommended value for <vendor-name><product-name>.
+
+```
+<insert configuration code snippet here>
+```
+
+#### Configure the IPsec transform set
+
+< insert the instructions for creating the IPsec transform set here.>
+
+```
+<insert configuration code snippet here>
+```
+
+#### Configure the IPsec static virtual tunnel interface (SVTI)
+
+< insert the instructions for creating the IPsec SVTI here. Below are some
+examples of parameters to set.>
+
+-  Adjust the maximum segment size (MSS) value of TCP packets going through a
+router as discussed in
+[MTU Considerations](https://cloud.google.com/vpn/docs/concepts/mtu-considerations)
+for Cloud VPN.
+
+```
+<insert configuration code snippet here>
+```
+
+### Configuring the dynamic routing protocol
+
+Follow the procedure in this section to configure dynamic routing for traffic
+through the VPN tunnel or tunnels using the BGP routing protocol.
+
+< insert the instructions for configuring dynamic routing here. Below are some
+examples of parameters to set.>
+
+BGP timers are adjusted to provide more rapid detection of outages.
+
+To advertise additional prefixes to GCP, < insert instructions here>.
+
+```
+<insert configuration code snippet here>
+```
+
+### Saving the configuration
+
+Follow the procedure in this section to save the on-premises configuration.
+
+< insert the instructions for saving the configuration here.>
+
+```
+<insert configuration code snippet here>
+```
+
+### Testing the configuration
+
+It's important to test the VPN connection from both sides of a VPN tunnel. For either side, make sure that the subnet that a machine or virtual machine is located in is being forwarded through the VPN tunnel.
+
+First, create VMs on both sides of the tunnel. Make sure that you configure the
+VMs on a subnet that will pass traffic through the VPN tunnel.
+
+-  Instructions for creating virtual machines in Compute Engine are located
+in the 
+[Getting Started Guide](https://cloud.google.com/compute/docs/quickstart).
+-  Instructions for creating virtual machine for <vendor-name><product-name>
```

Comment:
Same issue with the angle brackets on these two lines. Change them to &lt; and to &gt; to get the text inside them to show up on the rendered page.